packages/app/.eslintrc.js

```diff
@@ -17,5 +16,18 @@ module.exports = {
   },
   rules: {
     '@typescript-eslint/no-unnecessary-type-assertion': [0],
+    '@typescript-eslint/no-unsafe-assignment': [0],
+    '@typescript-eslint/no-unsafe-return': [0],
+    '@typescript-eslint/no-unsafe-call': [0],
+    '@typescript-eslint/no-unsafe-member-access': [0],
+    '@typescript-eslint/ban-types': [0],
+    '@typescript-eslint/restrict-template-expressions': [0],
+    '@typescript-eslint/explicit-module-boundary-types': [0],
+    "no-warning-comments": [1],
+    "react-hooks/exhaustive-deps": [
+      "warn", {
+        additionalHooks: "useRecoilCallback",
+      },
+    ]
```

Comment:
Should we add these to `@stacks/eslint-config`?

Reply:
Hm, I'm not sure. Some you could argue are better left enabled, and anything React should be an extended config, or something.